fastcrud/crud/fast_crud.py

```diff
@@ -1655,9 +1674,70 @@ async def update(
         filters = self._parse_filters(**kwargs)
         stmt = update(self.model).filter(*filters).values(update_data)
 
+        if returning:
+            stmt = stmt.returning(self.model.__table__.columns)  # type: ignore[call-overload]
```

Comment:
Let's expose an interface to return only specific columns here - returning(['field1', 'field2']) passed to method arguments, ideally with some match schema validation and option for selecting all.

Reply:
Alright, I have made some adjustments in a04834f. PTAL.